_data/toc.yaml

```diff
@@ -2964,13 +2964,21 @@ manuals:
 - sectiontitle: Docker Store
   section:
   - path: /docker-store/
-    title: Docker Store overview
-  - path: /docker-store/publish/
-    title: Publish content on Docker Store
-  - path: /docker-store/trustchain/
-    title: Docker Store Trust Chain
-  - path: /docker-store/faq/
-    title: Consumer FAQs
+    title: About Docker Store
+  - sectiontitle: Docker Store FAQs
+    section:
+    - path: /docker-store/customer_faq/
+      title: Customer FAQs
+    - path: /docker-store/publisher_faq/
+      title: Publisher FAQs
+  - sectiontitle: Publisher
```

Comment:
I think this title should be "Publish content" or something like that. Or I suppose Publisher Guide maybe. @johndmulhausen for opinion

Reply:
What do you think @KickingTheTV?

Reply:
@mistyhacks , what about `For Publishers`? This section covers more than content (and there are plans to add more sub-sections).